spec/controllers/questions_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+require 'pp'
+
+describe QuestionsController do
+
+  describe 'create new question' do
+    context "POST /questions" do
+      before :each do
```

Comment:
0.  Good start.
1.  use `let()`
2.  might want to try using FactoryGirl to generate these attributes for you.